roles/openshift_node/handlers/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-- name: restart openshift-node
-  service: name=openshift-node state=restarted
+- name: restart node
+  service: name={{ product_type }}-node state=restarted
```

Comment:
I'm not convinced that one knob is sufficient...  I can see demand (possibly imagined) for a differentiation between origin/atomic-enterprise and origin/openshift.